src/main/java/io/github/thebusybiscuit/sensibletoolbox/api/items/BaseSTBBlock.java

```diff
@@ -985,17 +1003,22 @@ private boolean attachLabelSign(PlayerInteractEvent event) {
         return true;
     }
 
-    private boolean placeLabelSign(Block signBlock, BlockFace face) {
+    private boolean placeLabelSign(Block signBlock, BlockFace face, @Nonnull Material signType) {
+        Validate.notNull(signType);
```

Comment:
Please add a message to the validation
```suggestion
        Validate.notNull(signType, "The Sign Type cannot be null");
```